src/Packagist/WebBundle/Entity/AuthorRepository.php

```diff
@@ -13,14 +13,14 @@
 namespace Packagist\WebBundle\Entity;
 
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
-use Symfony\Bridge\Doctrine\RegistryInterface;
+use Doctrine\Persistence\ManagerRegistry;
```

Comment:
`Symfony\Bridge\Doctrine\RegistryInterface` was deprecated, so changed to the base registry.